R/derive_vars_dy.R

```diff
@@ -84,29 +90,36 @@ derive_vars_dy <- function(dataset,
 
   if (length(bad_vars > 0)) {
     err_msg <-
-      sprintf(paste0("source_vars must end in DT or DTM or be explicitly and uniquely named.\n",
-                     "Please name or rename the following source_vars:\n", "%s"),
-              paste0(bad_vars, collapse = ", "))
+      sprintf(
+        paste0(
+          "source_vars must end in DT or DTM or be explicitly and uniquely named.\n",
+          "Please name or rename the following source_vars:\n", "%s"
+        ),
+        paste0(bad_vars, collapse = ", ")
+      )
     abort(err_msg)
   }
 
   dy_vars <- if_else(source_names == "",
-                     stringr::str_replace_all(vars2chr(source_vars), "(DT|DTM)$", "DY"),
-                     source_names)
+    stringr::str_replace_all(vars2chr(source_vars), "(DT|DTM)$", "DY"),
```

Comment:
should `source_names` on line 93 be on next line? to be aligned with other parameters to `if_else`

Reply:
Yes